common/include/pcl/experimental/executor/default/omp_executor.hpp

```diff
@@ -50,8 +50,8 @@ struct omp_executor {
   template <typename F>
   void bulk_execute(F&& f, shape_type n) const {
 #ifdef _OPENMP
-  #pragma omp parallel for num_threads(n)
-    for (int i = 0; i < n; ++i) std::forward<F>(f)(omp_get_thread_num());
+  #pragma omp parallel for
```

Comment:
The first line change might not be goof: https://www.openmp.org/spec-html/5.0/openmpse14.html#x54-800002.6

num_threads sets the shape. How are you setting the shape without it?

Reply:
This should be resolved once integrate the latest version of executors which has shape incorporated.